### PR TITLE
ci: add eslint, stylelint, airbnb-styleguide, prettier 

### PR DESCRIPTION
**Pull request by anttibull**
[CASH-11], [CASH-10], [CASH-11], [CASH-12]

[CASH-11]: https://cash69.atlassian.net/browse/CASH-11?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CASH-10]: https://cash69.atlassian.net/browse/CASH-10?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CASH-11]: https://cash69.atlassian.net/browse/CASH-11?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CASH-12]: https://cash69.atlassian.net/browse/CASH-12?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sashalavrus — DISMISSED** [review]

File: lefthook.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pre-commit:
+    commands:
+        eslint:
+            glob: '*.{ts,tsx}'
+            run: yarn lint
+        ts-lint:
+            glob: '*.{ts,tsx}'
+            run: yarn tsc --noEmit
+        stylelint:
+            glob: '*.{css}'
```
Comment (sashalavrus) [inline lefthook.yml:10]: scss, do wr need here?

File: package.json [hunk 2]
```diff
@@ -4,25 +4,19 @@
...
+    "chromatic": "chromatic --exit-zero-on-changes",
     "eject": "react-scripts eject",
-    "format": "prettier --write src",
-    "lint": "tsc --noEmit && eslint --fix src/**/*.{ts,tsx} src/**/**/*.{ts,tsx}",
+    "format": "yarn lint --fix",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "sort-package-json": "sort-package-json 'package.json'",
     "start": "react-scripts start",
+    "storybook": "start-storybook -p 6006 -s public",
+    "stylelint": "stylelint **/*.css",
```
Comment (sashalavrus) [inline package.json:15]: do we need scss here?

File: tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```
Comment (sashalavrus) [inline tsconfig.json:3]: in `eslintconfig` we have `target: es2021` but here `es2020`, is it ok

File: lefthook.yml (hunk 1 above)
Comment (anttibull) [reply lefthook.yml:10]: I'm not sure, because we not going to setup scss, but we'll can discuss this issues

File: tsconfig.json (hunk 3 above)
Comment (anttibull) [reply tsconfig.json:3]: Yes, thanks , i'll fix

**Review by parovozik787 — APPROVED** [review]

**Pull request merged**